var/spack/repos/builtin/packages/r-seurat/package.py

```diff
@@ -7,13 +7,21 @@
 
 
 class RSeurat(RPackage):
```

Comment:
I think this package is tied with `r-condop` as the slowest concretizing package. But this package has a lot of reverse dependencies we could add to make it an even better concretizer stress test.